static/src/javascripts/types/global.d.ts

```diff
@@ -88,19 +88,22 @@ interface PageConfig extends CommercialPageConfig {
 	nonRefreshableLineItemIds?: number[];
 	section: string;
 	isPaidContent: boolean;
-	isSensitive: boolean;
```

Comment:
Why did you remove this key?

Reply:
ah I should have left a note - it's duplicated on line 81